docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3.3'
+services:
+    web:
+        build: 
+            context: .
+            dockerfile: Dockerfile
```

Comment:
`docker-compose` will use naturally Dockerfile. I added this to make sure or if anyone would like to change it as an example.